### PR TITLE
DDF-2665: Added missing failsafe dependency

### PR DESCRIPTION
**Pull request by rzwiefel**
#### What does this PR do?
Adds a missing dependency 
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@glenhein @bdeining 

#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[IO](https://github.com/orgs/codice/teams/IO)
[Core APIs](https://github.com/orgs/codice/teams/core-apis)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@jlcsmith
@kcwire

#### How should this be tested? (List steps with links to updated documentation)
starting and ingesting a powerpoint with thumnails
#### What are the relevant tickets?
https://codice.atlassian.net/browse/DDF-2665

### CODE REVIEW

File: catalog/transformer/catalog-transformer-pptx/pom.xml [hunk 1]
```diff
@@ -153,22 +158,26 @@
...
                         </Import-Package>
                         <Embed-Dependency>
-                            catalog-core-api-impl,
-                            platform-util,
+	                        catalog-core-api-impl,
                             catalog-transformer-common,
-
+                            commons-io,
+                            commons-lang3,
+                            failsafe,
```
Comment (rzwiefel) [inline catalog/transformer/catalog-transformer-pptx/pom.xml]: This is the only added thing in these sections (all the other `Import-Package` and `Embed-Dependency` sections were just rearranged).

Also the Dependency on 141-145 was added.

Comment (jlcsmith) [inline catalog/transformer/catalog-transformer-pptx/pom.xml]: Can you double check the formatting?  The indention looks off..

Comment (rzwiefel) [inline catalog/transformer/catalog-transformer-pptx/pom.xml]: Fixed

File: catalog/admin/admin-poller-service/pom.xml [hunk 2]
```diff
@@ -64,6 +64,11 @@
             <artifactId>spock-shaded</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>net.jodah</groupId>
+            <artifactId>failsafe</artifactId>
+            <version>${jodah-failsafe.version}</version>
```
Comment (jlcsmith) [inline catalog/admin/admin-poller-service/pom.xml]: Probably should move this into dependencyManagement of the root pom

**Comment by jlcsmith** [comment]
@rzwiefel - can you check the itest failures from the bamboo build please?

**Comment by rzwiefel** [comment]
Issues resolved. Just need to run full build to make sure everything is happy.

**Comment by rzwiefel** [comment]
Full build passed :+1:

**Pull request merged**